howtos/how_to_request_cached_data.py

```diff
@@ -0,0 +1,139 @@
+            subscription_expression(str): Topic subscription expression.
+            cache_name(str): name of the solace cache to retrieve from
+            cache_access_timeout(int): Solace cache request timeout, in milliseconds
+
+        Raises:
+            InterruptedException: If any thread has interrupted the current thread
+            PubSubPlusClientException: If the operation could not be performed
+            IllegalStateException: If the service is not running
+        """
+        cached_message_subscription_request_for_latest_of_cached_or_live_messages = \
+            CachedMessageSubscriptionRequest.live_cancels_cached(cache_name,
+                                                                 TopicSubscription.of(subscriptionExpression),
+                                                                 cache_access_timeout)
+
+        correlation_id = 12345
+        correlation_completion_listener = MyCorrelationCompletionListener()
+
+        direct_receiver.request_cached(cached_message_subscription_request_for_latest_of_cached_or_live_messages,
+                                       correlation_id,
+                                       correlation_completion_listener)
+
+    def create_cached_subscription_with_more_options(direct_message_receiver: 'DirectMessageReceiver',
+                                                     subscription_expression: 'SubscriptionExpression',
+                                                     cache_name: str,
+                                                     cache_access_timeout: int,
+                                                     max_cached_messages: int,
+                                                     cached_message_age: int):
+        """
+        An example how to create and perform CachedMessageSubscriptionRequest to receive first cached
+        messages when available, followed by live messages. Additional cached message properties such
+        as max number of cached messages and age of a message from cache can be specified. Live messages
+        will be queues until the solace cache response is received.
+        Queued live messages are delivered to the application after the cached messages are delivered.
+        Args:
+            direct_message_receiver(DirectMessageReceiver): An already configured and connected receiver
+            subscription_expression(str): topic subscription expression
+            cache_name(str): Name of the solace cache to retrieve from
+            cache_access_timeout(int): Solace cache request timeout, in milliseconds
+            max_cached_messages(int): The max number of messages expected to be received from a Solace cache
+            cached_message_age(int): The maximum age, in seconds, of the messages to retrieve from a Solace cache
+            ignore_cache_access_exceptions(bool): `True` prevents subscription rollback on Solace cache request errors. `False` is the default value.
```

Comment:
`ignore_cache_access_exceptions` is not a flag for this method or something you can pass in to `cached_first`

Reply:
Please review fix